### PR TITLE
Feature/admin page

### PR DESCRIPTION
**Pull request by tenodi**

### CODE REVIEW

**Comment by tenodi** [comment]
Agreed - thanks! I've run it through both. However, I don't agree with using active over passive in our case, since CLI is making all the action of creating extension parts and that is something that is known. I'm using American English (e.g. customize).

P.S. Please review it by running it locally on your machine. That way you can check for markdown and highlighting errors which are important.

File: _posts/1-8-1-ShoutemCloud.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
```
Comment (MihaShoutem) [inline _posts/1-8-1-ShoutemCloud.md]: Run this post through http://www.grammarcheck.net/editor/. Grammarly reports no issues, which is strange.

File: _posts/1-8-3-DataSchemas.md [hunk 2]
```diff
@@ -0,0 +1,330 @@
...
+
+## Usage in extensions
+
+Create data schema with:
+
+```ShellSession
+$ shoutem schema add <schema-name>
+```
+
+where you should replace <schema-name> with the name of your Data Schema name. Example:
```
Comment (MihaShoutem) [inline _posts/1-8-3-DataSchemas.md]: <schema-name> is not visible.

File: _posts/1-8-3-DataSchemas.md [hunk 3]
```diff
@@ -0,0 +1,330 @@
...
+
+```ShellSession
+$ shoutem schema add Restaurants
+Schema `Restaurnats` is created in file `server/schemas/Restaurants.json`!
```
Comment (MihaShoutem) [inline _posts/1-8-3-DataSchemas.md]: Restaurnats -> Restaurants

File: _posts/1-8-3-DataSchemas.md [hunk 4]
```diff
@@ -0,0 +1,330 @@
...
+      "type": "string"
+    }
+  },
+  "titleProperty": "name",
+  "type": "object"
+}
+```
+
+Root JSON fields that are immediately included are:
+- `title`: Title of the schema shown on the CMS page
```
Comment (MihaShoutem) [inline _posts/1-8-3-DataSchemas.md]: I can't see a list here. It's rendered as text.

File: _posts/1-8-3-DataSchemas.md [hunk 5]
```diff
@@ -0,0 +1,330 @@
...
+```
+
+
+### Additional descriptor properties
+
+Value descriptor along with _value type_ can also describe additional information for particular value. These fields are inherited from JSON Schema specification:
+
+- `properties.pattern` - regex pattern constraint, applicable only to string primitive type
+- `properties.required` - required constraint
+- `properties.minLength` and `properties.maxLength` - string length constraints, applicable only to string primitive type
```
Comment (MihaShoutem) [inline _posts/1-8-3-DataSchemas.md]: Highlight 'string' somehow

File: _posts/1-8-4-ReduxIO.md [hunk 6]
```diff
@@ -0,0 +1,8 @@
+---
+layout: doc
+permalink: /docs/cloud/redux-io
+title: Introduction
+section: Shoutem Cloud
+---
+
+Coming soon
```
Comment (MihaShoutem) [inline _posts/1-8-4-ReduxIO.md]: Add margin above 'Coming soon'. It sticks to the top of the screen.

**Pull request merged**